docs/tutorials/cli.rst

```diff
@@ -198,9 +205,9 @@ To setup pseudolocalization add :conf:`pseudoLocale` in ``package.json``::
 :conf:`pseudoLocale` option can be any string 
 examples: :conf:`en-PL`, :conf:`pseudo-LOCALE`, :conf:`pseudolocalization` or :conf:`en-UK`
 
-PseudoLocale folder is automatically created based on configuration when running 
-``yarn extract`` command. Pseudolocalized text is created on  ``yarn compile`` command.
-The pseudolocalization is automatically created from default messages. 
+PseudoLocale string have to be in `locale` config as well. Otherwise no folder is going to be created.
+Pseudolocalized text is created on  ``yarn compile`` command.
+The pseudolocalization is automatically created from messages in order specified in "LINK TO Preparing catalogs for production".
```

Comment:
How can I link to added compile section above?

Reply:
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html

Reply:
Every heading is an anchor that you can reference with `:ref:`:

```suggestion
The pseudolocalization is automatically created from messages in order specified in :ref:`Preparing catalogs for production`.
```

You could also customize the link if you want to spell it differently:

```
The pseudolocalization is automatically created from messages in order specified in :ref:`section above<Preparing catalogs for production>`.
```